app/assets/stylesheets/setup-mode.scss

```diff
@@ -6,3 +6,14 @@ body.default-header {
   padding-top: 0;
   min-height: 100vh;
 }
+
+#page-content {
```

Comment:
We were repeating this css for the signup page, confirmation page and we needed it for this settings page as well. Hence it made sense to instead add it into the setup_mode css as it seems prevalent to the design for pages in setup mode.

Reply:
@nickytonline from my understanding this would not work, as we don't want this css rules applied to all pages, and #page-content forms as part of the shell.